src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Colors randomColor = Colors.valueOf(colorSupplier.getRandomColor());
+        int randomNumber = new Random().nextInt(100) + 1;
```

Comment:
no need to create new Random() each time when you want to use it, make it class-level field